azure/template.json

```
@@ -163,19 +170,17 @@
                                 "name": "Redis",
                                 "connectionString": "[parameters('loggingRedisConnectionString')]",
                                 "type": "Custom"
-                            },
-                            {
-                                "name": "PensionsRegulatorSql",
-                                "connectionString": "[variables('SqlConnectionstring')]",
-                                "type": "custom"
                             }
                         ]
                     },
                     "customHostName": {
                         "value": "[parameters('internalApiCustomHostname')]"
                     },
                     "certificateThumbprint": {
-                        "value": "[reference(resourceId(parameters('sharedAppServicePlanResourceGroup'), 'Microsoft.Web/certificates', parameters('internalApiCertificateName')), '2016-03-01').Thumbprint]"
+                        "value": "[reference('app-service-certificate').outputs.certificateThumbprint.value]"
```

"value": "[if(greater(length(parameters('internalApiCertificateName')), 0), reference('app-service-certificate').outputs.certificateThumbprint.value, '')]"
```suggestion
                        "value": "[reference('app-service-certificate').outputs.certificateThumbprint.value]"
```